pyoptsparse/pyOpt_optimizer.py

```diff
@@ -974,9 +975,9 @@ def OPT(optName, *args, **kwargs):
     ----------
     optName : str or enum
        Either a string identifying the optimizer to create, e.g. "SNOPT", or
-       an enum accessed via ``pyoptsparse.Optimizers``, e.g. ``Optimizers.SNOPT``.
+       an enum accessed via :class:`~pyoptsparse.pyOpt_optimizer.Optimizers`, e.g. ``pyoptsparse.Optimizers.SNOPT``.
 
-    \\*args, \\*\\*kwargs : varies
+    \*args, \*\*kwargs : varies
```

Comment:
I remember this was modified in #441 and then #444, what's the reason for reverting?

Reply:
Actually I take it back, it's not just a preference it is described in [PEP257](https://peps.python.org/pep-0257/#what-is-a-docstring):
```
Use `r"""raw triple double quotes"""` if you use any backslashes in your docstrings.
```